sandboxed_containers/deploying-sandboxed-container-workloads.adoc

```diff
@@ -12,31 +12,32 @@ include::modules/sandboxed-containers-preparing-openshift-cluster.adoc[leveloffs
 .Additional Resources
 * xref:../installing/installing_bare_metal_ipi/ipi-install-overview.adoc#ipi-install-overview[Deploying installer-provisioned clusters on bare metal]
 
-== Deploying {sandboxed-containers-operator} using the web console
+== Deploying {sandboxed-containers-first} workloads using the web console
 
-You can install the Operator and view your workloads from the web console.
+You can deploy {sandboxed-containers-first} workloads from the web console. First, you must install the {sandboxed-containers-operator}, then create the `KataConfig` custom resource. Once you are ready to deploy a workload in a sandboxed container, you must manually add the kata runtime to the workload's YAML file.
 
 include::modules/sandboxed-containers-installing-operator-web-console.adoc[leveloffset=+2]
+include::modules/sandboxed-containers-create-kataconfig-resource-web-console.adoc[leveloffset=+2]
+//include::modules/sandboxed-containers-selecting-nodes-web-console.adoc[leveloffset=+3]
+include::modules/sandboxed-containers-deploying-workloads-with-kata-runtime-web.adoc[leveloffset=+2]
 
-include::modules/sandboxed-containers-viewing-workloads-from-web-console.adoc[leveloffset=+2]
+== Deploying {sandboxed-containers-first} workloads using the CLI
 
-== Deploying {sandboxed-containers-operator} using the CLI
-
-You can install and deploy the Operator and view workloads from the CLI.
+You can deploy {sandboxed-containers-first} workloads using the CLI. First, you must install the {sandboxed-containers-operator}, then create the `KataConfig` custom resource. Once you are ready to deploy a workload in a sandboxed container, you must add the kata runtime to the workload's YAML file.
```

Comment:
Why not abbreviate custom resource?

Shouldn't this be `kata` and `RuntimeClass` as in some of the previous instances?

Reply:
Same comment as above.